arch/sparc/Kconfig

```diff
@@ -68,6 +68,9 @@ config NET_TX_STACK_SIZE
 	default 2048
 
 config NET_RX_STACK_SIZE
-	default 2048
+	default 3072
```

Comment:
architecture should never set stack sizes for subsystems like this, this is something that need to be set bt the application and not the architecture.

Reply:
OK I understand that the default should not be here. The background to putting the defaults here was to make the test suite pass. The SPARC ABI just requires some stack space and the defaults defined in the subsystems Kconfigs are just to low.

Would it be an acceptable solution to update the current defaults in the respective subsystem, conditioned on `CONFIG_SPARC`? (An alternative would be to update in (many) individual `prj.conf` but that would affect all other platforms aswell.)

Reply:
@martin-aberg: As I mentioned on similar occasion in another ticket, the problem here is that you're trying to add adhoc workaround just for SPARC. Whereas we (as "all people working on Zephyr") should look for generic solutions to issues we face.

> The SPARC ABI just requires some stack space

Do you mean *additional* stack space? Is it routed in windowed registers? Well, some Xtensa CPU configs also have windowed registers, so SPARC is not unique here. Way to handle that would be to add something like CONFIG_ADDITIONAL_STACK_SPACE (not literally, perhaps it should be CONFIG_ARCH_STACK_ADDEND, bikeshedding definitely needed), and then *add* that to all placed where stack size is mentioned. I.e. if there was:

```
uint8_t stk[CONFIG_STACK_FOO];
```

now it would be:

```
uint8_t stk[CONFIG_STACK_FOO + CONFIG_ARCH_STACK_ADDEND];
```

That's rough idea from 2 minutes of thinking (and not looking at source code), perhaps some another way would be better. It just shows the general direction of thinking how to support that requirement in generic manner.